plugins/osdn/multitenant/multitenant.go

```diff
@@ -31,6 +31,7 @@ func Master(osClient *osclient.Client, kClient *kclient.Client, clusterNetwork s
 	if err != nil {
 		glog.Fatalf("SDN initialization failed: %v", err)
 	}
+	kc.AdminNamespaces = append(kc.AdminNamespaces, "default")
```

Comment:
impact of this is that things in the default namespace have network visibility to all other namespaces? that seems like it should at least be optional